etherman/etherman.go

```diff
@@ -1295,18 +1296,25 @@ func decodeSequences(txData []byte, lastBatchNumber uint64, sequencer common.Add
 			return nil, err
 		}
 		coinbase := (data[1]).(common.Address)
+		dataAvailabilityMessage := (data[2]).([]byte) // TODO: is this right???
 		sequencedBatches := make([]SequencedBatch, len(sequencesValidium))
-		for i, seq := range sequencesValidium {
+		var batchNums []uint64
```

Comment:
⚠️  There's something super important that I forgot to explain! My bad 😅 Let's huddle later, just dropping this warning to not forget about this... tl;dr: 

DA only cares about non-forced batches, as forced batches are posted on-chain, relevant smart contract code [here](https://github.com/0xPolygonHermez/zkevm-contracts/blob/develop/contracts/v2/consensus/validium/PolygonValidiumEtrog.sol#L168)